### PR TITLE
Add readme for post call config

### PR DESCRIPTION
**Pull request by ralphgabrinao**
# What
New md file for post-call survey configuration

# Why
A dedicated section for post-call survey will make it easier to know the different configurations for post-call. Soon the md file will be expanded to include one question poll.

# How Tested

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: docs/post-call-survey.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+The [Azure Communication Services UI Library](https://azure.github.io/communication-ui-library/) provides a default post-call screen that is rendered after a call ends, no configuration required.
+
+### Configure using environment variables
+
+Remove `VV_POSTCALL_SURVEY_TYPE` from your environment variables.
+
+### Configure using defaultConfig.json
+
+Remove `postCall` from your config.
```
Comment (tejaswip-msft) [inline docs/post-call-survey.md]: there wont be any 'postCall' in the config in this case right, so can we just say that leave the default config as is

Comment (ralphgabrinao) [reply docs/post-call-survey.md]: That's true, though I think by saying 'remove' or 'delete' explicitly tells the user that postCall should not be set. Also would apply once one question poll is added.

Comment (tejaswip-msft) [reply docs/post-call-survey.md]: Yup makes more sense once one question poll is enabled

File: docs/post-call-survey.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                "surveyUrl": "<your_survey_url>"
+            }
+        }
+    }
+}
+```
+
+## Use the Default Post-Call Screen
+
+The [Azure Communication Services UI Library](https://azure.github.io/communication-ui-library/) provides a default post-call screen that is rendered after a call ends, no configuration required.
```
Comment (tejaswip-msft) [inline docs/post-call-survey.md:61]: is it okay that these links open in the same tab?

Comment (ralphgabrinao) [reply docs/post-call-survey.md:61]: I think so. That's also the case for the links in the main README. Apparently there is no way to open link in new tab in GitHub markdown: https://stackoverflow.com/a/42156438

**Review by tejaswip-msft — APPROVED** [review]

**Comment by ralphgabrinao** [comment]
@hansung-msft -- 1QP Will be added later when the feature is completed

**Review by hansung-msft — APPROVED** [review]

**Pull request merged**